Ansible/roles/kubernetes/tasks/kubernetes.yml

```diff
@@ -16,9 +18,11 @@
 
 - name: "Install kubernetes packages"
   dnf:
-    name: [ "kubeadm-{{ kubernetes_version }}", "kubelet-{{ kubernetes_version }}", "kubectl-{{ kubernetes_version }}" ]
+    name: >
+      ["kubeadm-{{ kubernetes_version }}", "kubelet-{{ kubernetes_version }}"
+      , "kubectl-{{ kubernetes_version }}"]
```

Comment:
I admit this is much easier to read. I went through the other files and haven't seen one which should also be changed in that way too. Please don't hesitate to tell me otherwise.